### ISSUE TITLE
Proximal humeral epiphysiolysis

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11012]
**Preferred term label:**
Proximal humeral epiphysiolysis

**Synonyms**

**Definition (free text, please give PubMed ID)**
Anteroposterior (AP) view of the shoulder in external rotation showing widening of the proximal humerus physis in comparison to contralateral shoulder with possible metaphyseal bony irregularity

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Proximal humeral epiphysiolysis

**Your nano-attribution (ORCID)**
https://radiopaedia.org/articles/little-league-shoulder

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001341
- label: Proximal humeral epiphysiolysis
- parents: [Abnormality of the proximal humeral epiphysis (HP:0010598)](https://hpo.jax.org/app/browse/term/HP:0010598)
- pmid: PMID:30485006
- def: Proximal humeral epiphysiolysis (Little League shoulder) is a shear or stress injury of the epiphyseal cartilage of the proximal humerus. Anteroposterior (AP) view of the shoulder in external rotation showing widening of the proximal humerus physis in comparison to contralateral shoulder with possible metaphyseal bony irregularity.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11012
- synonyms: Little league shoulder(NONE)
